### DISCUSSION TITLE
Documentation update

### DISCUSSION THREAD

**Discussion by dipcse07** [YueChen-C/py-ios-device#112] [Q&A] [answered]
In the documentation i can see there are few instructions for network monitoring (may be) but there is no proper documentation on how to import and use this library directly to use in python code to get the device info data or battery info data or any other info..

Can anyone please give that documentation for those apis so that as a noob in python i can also use this library in my project ?
@YueChen-C , @tomriddly  Can you  guys  :(  please check this out. 
Thanks in advance

**Comment by YueChen-C** [comment]
I will hurry up and add

**Reply by dipcse07** [reply]
@YueChen-C  thank you very much.. will be eagerly waiting for this update and work around with the apis directly... 👍

**Comment by YueChen-C** [comment, marked as answer]
You can have many references such as  https://github.com/YueChen-C/py-ios-device/tree/main/demo

**Reply by dipcse07** [reply]
Thank you...I got necessary battery info code from here  in the instrument demo: https://github.com/YueChen-C/py-ios-device/tree/main/demo/instrument_demo
